pkg/sql/schemachanger/scbuild/internal/scbuildstmt/process.go

```diff
@@ -11,31 +11,58 @@
 package scbuildstmt
 
 import (
+	"reflect"
+
+	"github.com/cockroachdb/cockroach/pkg/settings"
 	"github.com/cockroachdb/cockroach/pkg/sql/schemachanger/scerrors"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 )
 
+var declarativeForSupportedAllOps = settings.RegisterBoolSetting(
+	"sql.schema_changer.declarative_for_all",
```

Comment:
Can we come up with a better name than `declarative_for_all`? 

How about `declarative_enable_unsupported`? Although, this isn't that great either, and we can probably do better.